File: src/main/java/gg/mineral/practice/commands/stats/PotsCommand.java
package gg.mineral.practice.commands.stats;

import org.bukkit.Material;

import gg.mineral.practice.commands.PlayerCommand;
import gg.mineral.practice.entity.PlayerStatus;

import gg.mineral.practice.managers.ProfileManager;
import gg.mineral.practice.util.messages.impl.ChatMessages;
import gg.mineral.practice.util.messages.impl.ErrorMessages;
import lombok.val;

public class PotsCommand extends PlayerCommand {

	public PotsCommand() {
		super("pots");
		setAliases("potions");
	}

	@Override
	public void execute(org.bukkit.entity.Player pl, String[] args) {
		val profile = ProfileManager
				.getProfile(pl.getUniqueId(), p -> p.getPlayerStatus() == PlayerStatus.FIGHTING);

		if (profile == null) {
			pl.sendMessage(ErrorMessages.NOT_IN_MATCH.toString());
			return;
		}

		val pots = profile.getInventory().getNumber(Material.POTION, (short) 16421);
		ChatMessages.POTS.clone().replace("%pots%", "" + pots).send(pl);
	}
}

File: src/main/kotlin/gg/mineral/practice/commands/stats/PotsCommand.kt
package gg.mineral.practice.commands.stats

import dev.rollczi.litecommands.annotations.command.Command
import dev.rollczi.litecommands.annotations.context.Context
import dev.rollczi.litecommands.annotations.execute.Execute
import gg.mineral.practice.entity.PlayerStatus
import gg.mineral.practice.entity.Profile
import gg.mineral.practice.util.messages.impl.ChatMessages
import gg.mineral.practice.util.messages.impl.ErrorMessages
import org.bukkit.Material

@Command(name = "pots", aliases = ["potions"])
class PotsCommand {
    @Execute
    fun execute(@Context profile: Profile) {
        if (profile.playerStatus !== PlayerStatus.FIGHTING) return profile.message(ErrorMessages.NOT_IN_MATCH)

        val pots = profile.inventory.getNumber(Material.POTION, 16421.toShort())
        profile.message(ChatMessages.POTS.clone().replace("%pots%", pots.toString()))
    }
}
